AskUbuntu/dist/metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+"author": {
+"name": "Bünyamin Öztürk",
+"id": "487515609815580672"
+},
+"service": "AskUbuntu",
+"description": {
+"en": "Ask Ubuntu is a community driven question and answer website for the Ubuntu operating system. It is part of the Stack Exchange Network, running the same software as Stack Overflow. Members gain reputation based on the community's response to their questions and answers.",
+"tr": "Ask Ubuntu, Ubuntu işletim sistemi için  topluluk odaklı bir soru cevap web sitesidir. Stack Overflow ile aynı yazılımı çalıştıran Stack Exchange Network'ün bir parçasıdır. Üyeler, topluluğun sorularına ve cevaplarına verdiği yanıtlara dayanarak itibar kazanır."
+},
+"url": [
+"www.askubuntu.com",
+"askubuntu.com"
+],
+"version": "1.0.1",
+"logo": "https://cdn.sstatic.net/Sites/askubuntu/img/apple-touch-icon@2.png",
+"thumbnail": "https://cdn.discordapp.com/attachments/647557097701244953/651454166853615616/opera_wabv8Za7Xe.png",
```

Comment:
please use another website to upload the images to, not discords cdn

Reply:
done 👌